src/main/resources/logback.xml

```diff
@@ -2,7 +2,7 @@
 <configuration>
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
-            <pattern>%highlight(%-5level) %d{yyyy-MM-dd'T'HH:mm:ss.SSS} %yellow([%thread]) %blue(%logger{36}\(%file:%line\)) %msg%n</pattern>
+            <pattern>%highlight(%-5level) %d{yyyy-MM-dd'T'HH:mm:ss.SSS} %yellow([%thread]) %blue(%logger{36}\(%file:%line\)) %yellow(%X{prefix}) %msg%n</pattern>
```

Comment:
I think we should add a comment about the prefix/MDC thing that notes its usage in CheckMonitoredTrip.java

Reply:
Added in 6fd00a976c063c5b46e7254a995fbbb628b90445